community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -382,7 +385,7 @@
     public static final Setting<Level> store_internal_log_level = setting( "dbms.logs.debug.level",
             options( Level.class ), "INFO" );
 
-    @Description( "Database timezone." )
+    @Description( "Database timezone. This setting influences which timezone the logs use." )
```

Comment:
that's used not only for logs so description is incorrect; monitoring procedures using this setting as well for example